source/concepts/networking/instance-access-and-bastions.html.md.erb

```diff
@@ -15,10 +15,14 @@ This provides secure and auditable access to EC2s without the need to expose por
 
 ## Bastions
 
-For instances running older versions of Linux where the [SSM Agent](https://docs.aws.amazon.com/systems-manager/latest/userguide/ssm-agent.html) can't be installed, we can provide a [bastion host](https://en.wikipedia.org/wiki/Bastion_host).
+For instances running older versions of Linux where the [SSM Agent](https://docs.aws.amazon.com/systems-manager/latest/userguide/ssm-agent.html) can't be installed, we can provide a [bastion host](https://en.wikipedia.org/wiki/Bastion_host).   
 
 The bastion will be preconfigured with the relevant security and network connectivity required.
 You can then securely connect to this bastion host via Systems Manager, and then on to your instance.
 
+If you find the bastion is down (between 20:00 and 05:00) then you may need to restart it. The best way to do this is to amend the Auto Scaling Group called bastion_linux_daily to set the values to 1 where they are 0. This will build a bastion EC2 server.
```

Comment:
Could you specify which values to change please - (Desired capacity, Minimum capacity and Maximum capacity) thanks

Reply:
Change made and pushed through